src/VCR.ts

```diff
@@ -62,10 +62,13 @@ async function writeCassetteFile(cassettes: CassetteFile): Promise<void> {
 async function saveCassettes(defns: nock.Definition[]): Promise<void> {
   const cassettes: CassetteFile = await readCassetteFile()
   const testName = expect.getState().currentTestName
-  if (testName) {
-    cassettes[testName] = defns
-    await writeCassetteFile(cassettes)
-  }
+  if (!testName) return
+
+  const hasCasset = cassettes[testName] && !!cassettes[testName].length
```

Comment:
This is pretty hard to follow can you add a comment please.

Reply:
I changed the code and made it more readable